tests/functional/kata-deploy/kata-deploy.bats

```diff
@@ -73,6 +73,20 @@ setup() {
 	do
 		kubectl get runtimeclass | grep -E "${handler_re}"
 	done
+
+	# Ensure that kata-deploy didn't corrupt containerd config, by trying to get the container runtime and node status
+	echo "::group::kubectl node debug"
+	kubectl get node -o wide
+	kubectl describe node --selector='!node-role.kubernetes.io/control-plane'
```

Comment:
Maybe drop the selector since all nodes in the cluster can be control-plane ?

Reply:
Good point - sorry for my lack of consistency after I made the change in the wait. Will update now!